code/modules/projectiles/guns/ballistic/bows/bow_arrows.dm

```diff
@@ -168,3 +170,22 @@
 		return
 	to_chat(human_target, span_danger("[src] reacts with the flames enveloping you! Oh shit!"))
 	explosion(src, light_impact_range = 1, flame_range = 2) //ow
+
+/// Ashen arrows
+/obj/item/ammo_casing/arrow/ashen
+	name = "ashen arrow"
+	desc = "An arrow made from watcher sinew and bone. Seems unusually lethal against the creatures it is made from."
+	icon_state = "ashen_arrow"
+	inhand_icon_state = "ashen_arrow"
+	base_icon_state = "ashen_arrow"
+	projectile_type = /obj/projectile/bullet/arrow/ashen
+
+/// ashen arrow projectile
+/obj/projectile/bullet/arrow/ashen
+	name = "ashen arrow"
+	icon_state = "ashen_arrow_projectile"
+
+/obj/projectile/bullet/arrow/ashen/Initialize(mapload)
+	. = ..()
+	AddElement(/datum/element/bane, mob_biotypes = MOB_MINING, damage_multiplier = 2.5, added_damage = 0)
```

Comment:
`/obj/projectile/bullet/arrow` has 50 damage, bane multiplier of 2.5, so it's capable of one-shotting legions and bileworms.

Reply:
Just added a flat damage increase so that it interacts consistently regardless of the damage multiplier.